_topics/en/compact-block-filters.md

```diff
@@ -22,11 +22,13 @@ excerpt: >
 
 ## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
 extended_summary: |
-  A full node with a copy of a block and some related data uses
-  [BIP158][] to create a Golomb-Rice Coded Set (GCS) of data from that
-  block.  The GCS (called a filter) is then distributed to wallets using either
-  the P2P protocol as described in [BIP157][] or some other mechanism, allowing
-  the wallet to search for any matches to their scripts.  If a match
+  A full node with a copy of a block and the data necessary to validate
+  that block uses [BIP158][] to create a Golomb-Rice Coded Set (GCS) of
+  data from that block (such as the prevout scriptPubKey from each input spent
+  in the block and the scriptPubKey for each output created in the
+  block).  The GCS (called a filter) is then distributed to wallets
+  (such as via the P2P protocol as described in [BIP157][]), allowing
+  them to search for any matches to their scripts.  If a match
```

Comment:
@jnewbery this commit confuses me; why did you restore the *prevout* terminology after arguing against it?